src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -7,10 +7,10 @@
     "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/bigquery.svg",
     "clients": [
       {
+      "name":"Google Cloud Java Client for BigQuery",
       "language":"java",
       "site":"https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-install-java",
-      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/0.22.0/apidocs/com/google/cloud/bigquery/package-summary.html",
-      "version" : "0.22.0",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/bigquery/package-summary.html",
       "infotip":"The com.google.cloud.bigquery packages",
```

Comment:
This might be useful to encode in the metadata to enable UI discovery (e.g., a quick-fix on unresolved symbols)

Reply:
can you elaborate? I'm not sure exactly what you're proposing here.

Reply:
Expressing this as:
```
"javaPackages":["com.google.cloud.bigquery", "com.google.cloud.bigquery.**"],
```

For example, PDE contributes a quick-fix to suggesting adding an `Import-Package` or `Require-Bundle` on an unresolved symbol that is found in a bundle:

![screen shot 2017-08-28 at 4 22 18 pm](https://user-images.githubusercontent.com/202851/29792034-1a702564-8c0d-11e7-8ea0-c5153da2e9a8.PNG)

Reply:
That would be something different. This is human readable text for the infotip on the library.

Reply:
It would be good to have this in a machine readable format too for the tooling reasons above.

Maybe it would help to add additional information here.  For example, there are sometimes alpha versions of libraries that users may want to try.  Or if this is the Google API Client library, we could add a pointer to the Google Client library?

Reply:
Sounds good. How do we tell Eclipse about our packages when adding a library?

Reply:
I think we'd have a quick-fix on an unresolved symbol like [PDE's quick-fix](http://git.eclipse.org/c/pde/eclipse.pde.ui.git/tree/ui/org.eclipse.pde.ui/plugin.xml?h=master#n1656).  Our quick-fix would iterate on the library definitions to see if it can find a match.

Reply:
oops, ping on this request too.

Reply:
done